src/styles/blocks/stars--4.css

```diff
@@ -0,0 +1,44 @@
+.stars--4 {
+  display: flex;
+  margin-top: 4px;
```

Comment:
Be consistent with your margins. It's recommended to add only top or bottom margins, but not both, to avoid potential margin collapse issues.